postv3/postv3/settings/__init__.py

```diff
@@ -0,0 +1 @@
+from .local import *
```

Comment:
You need three more things:

1. Modify this as:
```
try:
    from .local import *
except ImportError:
    print('You need to create a local.py file. local_example.py is provided as an example')
```
2. Create a local_example.py ... with `from .prod import *` should be enough

3. Add local.py file to .gitignore